cyber.latex

```diff
@@ -0,0 +1,264 @@
+\documentclass[$if(fontsize)$$fontsize$,$endif$$if(lang)$$babel-lang$,$endif$$if(papersize)$$papersize$paper,$endif$$for(classoption)$$classoption$$sep$,$endfor$]{$documentclass$}
```

Comment:
Could you find a better color scheme for the markdown code syntax highlighting.  Light blue on white will not print out well.